python/pyspark/ml/wrapper.py

```diff
@@ -190,8 +190,30 @@ def _transform(self, dataset):
         return DataFrame(self._java_obj.transform(dataset._jdf), dataset.sql_ctx)
 
 
+class JavaCallable(object):
```

Comment:
I agree with having JavaWrapper inherit from JavaCallable.  Renaming sounds reasonable, but doing it in another PR sounds like a good idea.